.storybook/main.js

```diff
@@ -1,7 +1,9 @@
 module.exports = {
   "stories": [
     "../stories/**/*.stories.mdx",
-    "../stories/**/*.stories.@(js|jsx|ts|tsx)"
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)",
```

Comment:
These are not tailwind related changes as the commit message mentions. Can go as a separate commit

Reply:
Similarly all other changes in this commit that is not tailwind related can go as part of another commit

Reply:
okay , will look into it.